odooshow/odooshow.py

```diff
@@ -107,8 +107,12 @@ def _record_url(self, record):
         """Return a formatted link for relational records. Only supported terminals
 
         :param record record: Odoo record
-        :return str: Formatted url
+        :return str: Formatted url or None
         """
+
+        if not hasattr(record, "get_base_url"):
+            return None
```

Comment:
I'd fallback to `self.env['ir.config_parameter'].sudo().get_param('web.base.url')` before returning `None`

Reply:
What a sausage, I really did it too fast, You're right, I will update my PR